### ISSUE TITLE
Project: Make map of favorite LA meetups

### ISSUE THREAD

**Issue by machikoyasuda** [fullstackla/pairing-meetup#55]
Here is some documentation/tutorials/starter-template to follow: https://github.com/maptimeLA/git_your_map_here

This will help you:
1. Continue practicing going back and forth between Terminal, Chrome and Atom
2. Continue practicing Git commands (git status)
3. Learn how to run a server
4. Write some JavaScript

**Comment by machikoyasuda** [comment]
@leighphan feel free to dig into this issue if you're interested!

**Comment by IanDCarroll** [comment]
all right, I can chew on this for a little bit! :grin: 
@leighphan I don't have any answers, but I'd be happy to share the problem with you!

**Comment by IanDCarroll** [comment]
Git your map here has a few dependencies that need to be installed first. 
1. Git - likely you have it.
2. Sublime or Atom text editor. My guess is any text editor will do, so I'll try it with Vim or GVim.
3. Python 2.7 I'm pretty sure this comes standard on Mac. Also on Ubuntu.
4. QGIS 
- this I'm not sure I have, but here's a general install page: https://www.qgis.org/en/site/forusers/download.html
- and this is an easy install guide for Ubuntu Linux. Just copy the commands to the CLI and hit ENTER one-by-one. http://gis.stackexchange.com/questions/133033/installing-the-latest-qgis-version-on-ubuntu-14-04

**Comment by machikoyasuda** [comment]
You do not need to download QGIS or Python. You have all the tools already to make a JS map.

**Comment by IanDCarroll** [comment]
So I got git_your_map_here running by it's lonesome.
Next step (probably tomorrow) figure out how to integrate it with the HTML and CSS we've got going on already.

![screenshot from 2016-07-14 13 06 30](https://cloud.githubusercontent.com/assets/14797009/16854299/2e765c30-49c5-11e6-9210-5c83b6bd92b3.png)

![screenshot_20160714-125924](https://cloud.githubusercontent.com/assets/14797009/16853997/f1d4e70c-49c3-11e6-9927-3cedc6746385.png)

**Comment by machikoyasuda** [comment]
Here's a mini-challenge: Try to change the map layer: http://leaflet-extras.github.io/leaflet-providers/preview/

**Comment by IanDCarroll** [comment]
I've got the google spreadsheet with addresses.
I've installed mapbox geo-google into the google spreadsheet, and it appears to work.
I've got a mapquest api key and pointed it to the spreadsheet's url.
But when I try to run it, it stays stuck on this screen:
![screenshot from 2016-07-27 20 24 48](https://cloud.githubusercontent.com/assets/14797009/17200403/c3c3aaee-5439-11e6-85f0-52caa21bec20.png)

Does it just take a long time to get the data, or is there something obstructing the conversion?

went ahead and opened an issue on this: https://github.com/mapbox/geo-googledocs/issues/40
